src/platform/forms-system/src/js/definitions/address.js

```diff
@@ -174,16 +177,12 @@ export function uiSchema(
       if (stateList) {
         // We have a list and it’s different, so we need to make schema updates
         if (addressSchema.properties.state.enum !== stateList) {
-          const withEnum = _.set(
+          const withEnum = set(
             'state.enum',
             stateList,
             schemaUpdate.properties,
           );
-          schemaUpdate.properties = _.set(
-            'state.enumNames',
-            labelList,
-            withEnum,
-          );
+          schemaUpdate.properties = set('state.enumNames', labelList, withEnum);
```

Comment:
And how about this?

```suggestion
          schemaUpdate.properties.state.enum = stateList;
          schemaUpdate.properties.state.enumNames = labelList;
```

Reply:
Yeah this one is weird, we end up mutating `schemaUpdate.properties` in the 2nd `set`call anyway, so I'm not sure why `set` was even used here. I don't see any reason why this wouldn't work. I ran unit tests locally and everything passed okay.

Reply:
I applied this change in a new commit ✅